spyder/plugins/editor/widgets/codeeditor/codeeditor.py

```diff
@@ -4460,20 +4500,23 @@ def mouseMoveEvent(self, event):
 
         pos = event.pos()
         self._last_point = pos
-        alt = event.modifiers() & Qt.AltModifier
-        ctrl = event.modifiers() & Qt.ControlModifier
 
-        if alt:
-            self.sig_alt_mouse_moved.emit(event)
+        modifiers = event.modifiers()
+
+        if modifiers == self.mouse_shortcuts['jump_to_position']:
+            self.sig_scrollflag_shortcut_move.emit(event)
             event.accept()
             return
 
-        if ctrl:
+        if modifiers == self.mouse_shortcuts['goto_definition']:
             if self._handle_goto_uri_event(pos):
                 event.accept()
                 return
 
-        if self.go_to_definition_enabled and ctrl:
+        if (
+                self.go_to_definition_enabled and
+                modifiers == self.mouse_shortcuts['goto_definition']
```

Comment:
```suggestion
            self.go_to_definition_enabled and
            modifiers == self.mouse_shortcuts['goto_definition']
```

Fix Black formatting.